sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -458,6 +458,7 @@ case class StringReplace(srcExpr: Expression, searchExpr: Expression, replaceExp
 object Overlay {
 
   def calculate(input: UTF8String, replace: UTF8String, pos: Int, len: Int): UTF8String = {
+    require(pos > 0, "If you specify `pos`, it must be a positive whole number.")
```

Comment:
This change isn't related to this pr, right?

Reply:
Yes.

Reply:
I will remove it.